src/components/@shared/FormInput/InputElement/index.tsx

```diff
@@ -95,7 +99,11 @@ const InputElement = forwardRef(
               (sortedOptions as string[]).map(
                 (option: string, index: number) => (
                   <option key={index} value={option}>
-                    {option} {postfix}
+                    <Option
+                      option={option}
+                      prefix={prefixes?.[index]}
+                      postfix={postfixes?.[index]}
```

Comment:
question: is `actions` missing here?

Reply:
Yes, but that's on purpose since a link would not be clickable inside the `select` component.